segmentation_models_pytorch/encoders/timm_efficientnet.py

```diff
@@ -41,9 +43,8 @@ def get_efficientnet_kwargs(channel_multiplier=1.0, depth_multiplier=1.0, drop_r
         block_args=decode_arch_def(arch_def, depth_multiplier),
         num_features=round_channels(1280, channel_multiplier, 8, None),
         stem_size=32,
-        channel_multiplier=channel_multiplier,
+        round_chs_fn=partial(round_channels, multiplier=channel_multiplier),
         act_layer=Swish,
-        norm_kwargs={},  # TODO: check
```

Comment:
There is no more `norm_kwargs ` and `channel_multiplier` options. 
`norm_kwargs` was not used, so I just delete it, but in the case of `channel_multiplier`, you can still pass it shrough the new `round_chs_fn` argument. 

See https://github.com/rwightman/pytorch-image-models/commit/c4f482a08b369e00a27bbfd266781c98fde45016#diff-27c2bbd967991cbb5264f93cb5da34895fdab02424b2cc8c63d3d0768e65d47aR490